tailwind.config.js

```diff
@@ -15,4 +15,10 @@ module.exports = {
     preflight: false,
   },
   plugins: [],
+  safelist: [
+    {
+      pattern:
+        /(bg|border|rounded|shadow|h|w|m|p|font-family|font-weight|text|leading)-.*/,
```

Comment:
Note that most of these are safe from creating unused utilities. For example, `p-SUFFIX` utils will be generated, but not `pl-SUFFIX`

<img width="754" alt="image" src="https://user-images.githubusercontent.com/34709943/222026992-a42809de-93b3-49c5-8890-6bd47e5bf618.png">

One exception is the border utils though. With this regex, `border-y-SUFFIX`, `border-t-SUFFIX` etc will be generated even though they aren't used in our docs.
It would be good to exclude these somehow, but I haven't managed to find a regex expression that can:
- Exclude `border-[y|x|t|l|r|b]-SUFFIX`
- Match `border-SUFFIX`

It's not the end of the world adding in colors for 6 more prefixes, but if anyone is a regex wizard let me know

Reply:
So you should be able to do with with the [block list](https://tailwindcss.com/docs/content-configuration#discarding-classes), though that may mean we can never generate them so no sure how worth it that is